lib/endpoints/class-acf-to-rest-api-controller.php

```diff
@@ -229,6 +236,24 @@ protected function get_fields( $request, $response = null, $object = null ) {
 			return apply_filters( 'acf/rest_api/' . $this->type . '/get_fields', $data, $request, $response, $object );
 		}
 
+		protected function add_relation_fields( &$acf_data, $field ) {
+			foreach ( $acf_data as $k => $rel_array ) {
```

Comment:
@airesvsg, do you mean I have to take care here? Taking care = checking `$acf_data` for being an object?